src/coreclr/tests/src/CoreMangLib/system/type/typemakearraytype1.cs

```diff
@@ -1,238 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-using System;
-
-/// <summary>
-/// MakeArrayType()
-/// </summary>
-public class TypeMakeArrayType1
-{
-    #region Public Methods
-    public bool RunTests()
-    {
-        bool retVal = true;
-
-        TestLibrary.TestFramework.LogInformation("[Positive]");
-        retVal = PosTest1() && retVal;
-        retVal = PosTest2() && retVal;
-        retVal = PosTest3() && retVal;
-        retVal = PosTest4() && retVal;
-        retVal = PosTest5() && retVal;
-
-        return retVal;
-    }
-
-    #region Positive Test Cases
-    public bool PosTest1()
-    {
-        bool retVal = true;
-
-        TestLibrary.TestFramework.BeginScenario("PosTest1: Call MakeArrayType for reference type");
-
-        try
-        {
-            Type type = typeof(String);
-            Type arrayType = type.MakeArrayType();
```

Comment:
did we lose some coverage? eg `typeof(string).MakeArrayType();`. You're using mostly int.

Reply:
Ah yes these are in #36604. Should I extract? Some of those tests are failing on mono so I wasn’t sure

Reply:
Oh either PR is fine. But we should not hold on adding a test that fails on mono - just use the attribute to disable it with an issue. They are fixing many of these.

Reply:
https://github.com/dotnet/runtime/pull/36604 already contains a fix, so i'm happy keeping things there

Reply:
Planning on rebasing that PR once this one is merged in